### PR TITLE
Canary Deployment for Greeting Function

### PR DESCRIPTION
**Pull request by allinwonder**
*Description of changes:*

implemented blue/green deployment for greeting function

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: apps/lambda/greeting-sam.yaml [hunk 1]
```diff
@@ -9,20 +9,18 @@ Resources:
       Handler: org.aws.samples.compute.greeting.GreetingHandler::handleRequest
       Runtime: java8
       CodeUri: ../../../microservices-greeting/target/greeting.zip
-      Description: >-
-        Amazon API Gateway.
       MemorySize: 1024
       Timeout: 20
       Tracing: Active
-      Policies:
```
Comment (arun-gupta) [inline apps/lambda/greeting-sam.yaml]: Are these policies no longer required? How would the `greeting` service put value to Parameter Store?

Comment (allinwonder) [reply apps/lambda/greeting-sam.yaml]: This is more like an accident when I copy the canary setting into the template.

**Comment by allinwonder** [comment]
@jicowan I am still testing the setup, there are some IAM errors in the SAM template. Can you revert the merge? Sorry for the inconviniance

**Comment by arun-gupta** [comment]
@allinwonder looks good.

can you also update `readme.adoc` with the instructions?

**Comment by allinwonder** [comment]
@arun-gupta yep, readme.adoc is updated.

**Comment by jicowan** [comment]
@allinwonder sorry for missing this.  Do you still need me to revert the merge?

**Pull request merged**